gtk4/src/subclass/editable.rs

```diff
@@ -156,3 +156,15 @@ unsafe extern "C" fn editable_set_selection_bounds<T: EditableImpl>(
         end_position,
     )
 }
+
+pub unsafe trait EditableClassSubclassExt: ClassStruct {
```

Comment:
This probably needs special subclassing integration

Reply:
Isn't that enough for install_properties? we probably need another trait that has the delegate_[get|set]_property methods that's auto-implemented for EditableImpl though.

Reply:
Well, try it and let me know :) I'm a bit worried about users having to pass in the property id offset correctly there, and also it has to be called *after* the normal properties are installed.

Reply:
That's definitely not great indeed. I will create a custom editable and play around with the subclassing code to figure out something nice to use if possible :-)

Reply:
Maybe simply by checking `<Self as ObjectImpl>::PROPERTIES.len()` but OTOH that's not very composeable then. It would only work if there is one such function like in `GEditable` for adding more properties.

Or you could check with `g_object_class_list_properties()` how many are already installed at this point.

In any case, I think this function should be automatically called from `<Editable as IsImplementable>::interface_init()`